arviz/plots/autocorrplot.py

```diff
@@ -88,6 +90,16 @@ def plot_autocorr(
         max_lag = min(100, data["draw"].shape[0])
 
     plotters = list(xarray_var_iter(data, var_names, combined))
+    max_plots = rcParams["plot.max_subplots"]
```

Comment:
if "plot.max_subplots" is deleted, this will throw an exception

max_plots = rcParams["plot.max_subplots"]

Maybe use `.get` so it will return None

max_plots = rcParams.get("plot.max_subplots")

Reply:
rcParams keys cannot be deleted. You would get the exception before when trying to delete it

Reply:
ok, good